src/main/java/io/stargate/sgv2/jsonapi/service/embedding/operation/EmbeddingProviderFactory.java

```diff
@@ -41,7 +45,7 @@ private synchronized EmbeddingProvider addService(
 
     if (configuration.serviceProvider().equals(ProviderConstants.CUSTOM)) {
       try {
-        Optional<Class<?>> clazz = configuration.clazz();
+        Optional<Class<?>> clazz = configuration.implementationClass();
         if (clazz.isPresent()) {
           final EmbeddingProvider customEmbeddingProvider =
               (EmbeddingProvider) clazz.get().getConstructor().newInstance();
```

Comment:
I know this is existing code, not changed here, but I think there are a few exceptions this code could throw (either due to not having default constructor, or constructor throwing exception).
Might make sense to catch `Exception`, throw explicit `JsonApiException` to indicate internal/configuration problem.

Reply:
@tatu-at-datastax The code uses `custom` type just to do IT. There is one implementation which returns fixed vector values in the response. Unless the manually download this code and create a custom code, user cannot use this type. Since all the providers are Rest based should I remove this and create a Rest stub to do IT.

Reply:
I guess whatever exists is fine and we can always improve things if and when necessary.